book/localization.rst

```diff
@@ -49,6 +49,11 @@ accepting new content. Afterwards nobody will have any permissions on this
 locale, so make sure that you add this permission in the permissions tab of
 the contacts.
 
+.. note::
+
+    When adding new localizations check the configuration of the ``urls`` in the 
```

Comment:
To check for what? This note confuses me more than it helps :thinking:

Reply:
My guess would be that you are referring to the latest slack message, where only a single language was configured and therefore the other weren't accessible. Would describe that in a different manner, maybe something like this:

```
When adding a new localization you have to make sure that the localization is also covered by one of the webspaces URLs. This can either happen by using the `language` and `country` attributes of the `url` tag, or by specifying e.g. the `{localization}` placeholder. For more information see :doc:`webspace`.
```

And is there a way to directly jump to the "urls" anchor? :thinking:

Reply:
@danrot couldn't find a way in rst to jump to a anchor just when creating a normal link referencing to the webspaces`.html` file.